backend/src/sentiment/sentiment.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Dict
+import spacy
+from spacytextblob.spacytextblob import SpacyTextBlob
+
+# Takes ~1 min to run on 100 reviews.
```

Comment:
Something worth investigating that might speed up performance: what if you call `sentiment()` twice with 2 sets of reviews (e.g. a set of 1 star reviews and a different set of 5 star reviews). Generally, it's easier/faster to order when there are fewer options, but not 100% if that's applicable here too. What I'd have in mind for this is something like:

```
# In bestbuy_scraper.py:
reviews_5star = reviews.filter(...)
reviews_1star = reviews.filter(...)
sentiment(reviews_5star, "top_positive")
sentiment(reviews_5star, "top_negative")


# In sentiment.py:
def sentiment(reviews: List[Dict], type: str) -> Dict:
    ...

    selected_sentiment = max(sentiments) if type == "top_positive" else min(sentiments)
    index = sentiments.index(selected_sentiment)

    return {
        type: {
            "title": reviews[index].get("title"),
            "text": reviews[index].get("comment"),
            "rating": reviews[index].get("rating"),
        },
```

Reply:
It's slower to do it this way than using the existing method and filtering on 1 and 5 star reviews. So I'll just add the filter.

Reply:
Hmm that's surprising. Thanks for checking and sorry for wasting your time trying to implement it. I'm good with keeping it as is